CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 ## Next release
 
+## 0.41.0 / 2020-07-29
+
+* [CHANGE] Configmap-reload: Update to v0.4.0 (#3334)
+* [CHANGE] Update prometheus compatibility matrix to v2.19.2 (#3316)
+* [FEATURE] Add Synthetic Probes support (#2832)
+* [FEATURE] Support Prometheus vertical compaction (#3281)
+* [ENHANCEMENT] Propagate Prometheus image version to statefulset (#3358)
```

Comment:
nit: this is a part of consolidating image url logic.

Reply:
unified with 3103 enhancement (image logic consolidation)